exporter/collstats_collector.go

```diff
@@ -20,23 +21,36 @@ func (d *collstatsCollector) Describe(ch chan<- *prometheus.Desc) {
 }
 
 func (d *collstatsCollector) Collect(ch chan<- prometheus.Metric) {
+	ctx := context.Background()
```

Comment:
why don't use this? https://github.com/Percona-Lab/mnogo_exporter/pull/11/files#diff-a7b5017c3776f04bf91dede50897600fR14
Why do we need it at all?

Reply:
Created https://jira.percona.com/browse/PMM-6254 to track context/timeouts

Reply:
But that should be `context.TODO()`